mod/message.php

```diff
@@ -165,11 +165,23 @@ function message_content(App $a)
 
 		$cmd = $a->argv[1];
 		if ($cmd === 'drop') {
+			$r = DBA::SelectFirst('mail', ['convid'], ['id' => $a->argv[2], 'uid' => local_user()]);
```

Comment:
The exact method name is `selectFirst` without the first letter capitalized.
Additionally, please use a meaning name for the variable, I personally use the table name + eventual plural if needed, but with `selectFirst` you don't, so I would use `$mail` instead of `$r`.